cmd/podman/ps.go

```diff
@@ -288,7 +300,77 @@ func psCmd(c *cli.Context) error {
 		outputContainers = []*libpod.Container{latestCtr}
 	}
 
-	return generatePsOutput(outputContainers, opts)
+	pss := shared.PBatch(outputContainers, 8, opts)
+	if opts.Sort != "" {
+		pss, err = sortPsOutput(opts.Sort, pss)
+		if err != nil {
+			return err
+		}
+	}
+
+	// If quiet, print only cids and return
+	if opts.Quiet {
+		return printQuiet(pss, opts)
+	}
+
+	if opts.Format != "" {
+		if opts.Format == "json" {
+			return dumpJSON(pss)
+		}
+		return printFormat(opts.Format, pss, opts)
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
```

Comment:
This would be a lot clearer with comments - this part prints the headers, the next part prints each container, etc

Reply:
sure, done